.github/workflows/release-binaries.yml

```diff
@@ -0,0 +1,172 @@
+name: Deploy binaries and create release
```

Comment:
Does it attach binaries to a created release or do you need to push a tag so it creates the release?

Is there a way to have it do the first one, such that releases can be created manually on the Github UI?

Reply:
I try to change that. The action that is currently used creates a release and attaches the files. I'm sure there is an action that is able to do that.